content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -10,35 +10,25 @@ weight: 101
 
 ## v2.0.7 [2021-TK]
 
-### SQLite Metadata Store
-
-This release adds an embedded SQLite database
-for storing metadata required by the latest UI features like notebooks and annotations.
-
 ### Features
-- Add Geo graph type to be able to store in Dashboard cells. <!-- https://github.com/influxdata/influxdb/pull/19811:  -->
-- Add the properties of a static legend for line graphs and band plots. <!-- https://github.com/influxdata/influxdb/pull/21218:  -->
-- Remove feature flags for permanent UI features: <!-- https://github.com/influxdata/influxdb/pull/21531:  -->
-  `axisTicksGenerator`, `legendOrientation`, `mosaicGraphType`, and `bandPlotType`.
-- Allow hiding the tooltip independently of the static legend. <!-- https://github.com/influxdata/influxdb/pull/21547:  -->
-- Support pagination when listing users via the API. <!-- https://github.com/influxdata/influxdb/pull/21367:  -->
-- Add `influxd` configuration flag [`--sqlite-path`](/influxdb/v2.0/reference/config-options/#sqlite-path).
-  for specifying a user-defined path to the SQLite database file. <!-- https://github.com/influxdata/influxdb/pull/21543:  -->
-- Update `influxd` configuration flag `--store` to work with string values `disk` or `memory`.
-  `memory` continues to store metadata in-memory for testing;
-  `disk` persists metadata to disk via bolt and SQLite. <!-- https://github.com/influxdata/influxdb/pull/21543:  -->
+
+- Upgrade Flux to v0.117.0 <!-- https://github.com/influxdata/influxdb/pull/21519 -->
+- Optimize [table.fill()] execution within Flux aggregate windows. <!-- https://github.com/influxdata/influxdb/pull/21519 -->
+- Upgrade UI to v2.0.7 <!-- https://github.com/influxdata/influxdb/pull/21564 -->
```

Comment:
@pierwill, let's confirm with @wbaker85, but looks like the UI upgrade basically fixes a handful of UI issues introduced in 2.0.6...like autorefresh works again. Is it helpful to call this out as new feature or call out bug fixes this resolves?

Reply:
I had figured we would handle the UI updates similar to the flux updates. There are a couple feature-ish things with the new UI, like the improved version numbering for the login page, and the new `flux-lsp-browser`.

For flux upgrades we list it in the feature section, so it would make sense to me to treat the UI upgrade in the same way.

Reply:
Makes sense, thanks @wbaker85 @pierwill...perhaps we list explicitly what's improved with version number on the login page, update to `flux-lsp-browser` <version>, others?

Reply:
I'm not sure how we'll want to handle this going forward - we don't explicitly list everything that changes with the flux upgrades, but I guess that's a little less obvious to users.

For `2.0.7`, we could list out the features & bug fixes on the `v2.0.7` UI release page: https://github.com/influxdata/ui/releases/tag/OSS-v2.0.7. There's only 3 things so it's pretty simple.